generators/deployment/templates/toolchain_master.yml

```diff
@@ -13,7 +13,7 @@ toolchain:
 repo:
   service_id: hostedgit
   parameters:
-    repo_url: "{{tag 'zip_url'}}"
+    repo_url: "{{tag '#zip_url'}}{{tag 'zip_url'}}{{tag '/zip_url'}}{{tag '^zip_url'}}{{tag 'repository'}}{{tag '/zip_url'}}"
     repo_name: "{{tag 'toolchain.name'}}"
```

Comment:
Some of these may be escaped by Handlebars, so need a test to check the output is as expected. To stop Handlebars from escaping use triple braces : {{{   }}}